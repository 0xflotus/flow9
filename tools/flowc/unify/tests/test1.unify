import maybe;

ref None<α1> c= α0;

α3 c= [α2];
