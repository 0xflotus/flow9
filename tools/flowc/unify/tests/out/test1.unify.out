α0=ref {None<α1> .. *}
α3=[α102]
α100={None<α1> .. *}
α101={None<α1> .. *}
